Mac cmake errors

I bypassed the openAl in the install script and have everything else installed. However, when I attempt to do the cmake command I'm getting an error.

cmake -DGPU_BACKEND_CUDA=OFF -DGPU_BACKEND_OPENCL=ON  -DQt5_DIR=~/Qt/qt_/5.9.8/clang_64/lib/cmake/Qt5 -DCMAKE_BUILD_TYPE=Release -G Ninja ../../stitchEm

-- Studio version: Studio-v0.0.9-31-g20693a5-master.2019-08-29
CMake Error: The following variables are used in this project, but they are set to NOTFOUND.
Please set them or make sure they are set and tested correctly in the CMake files:
librtmp
    linked by target "rtmpPlugin" in directory /Users/michaelweiland/Documents/Development/stitchEm/IO/src/rtmp

-- Configuring incomplete, errors occurred!

I've attached the error log.

Macbook Pro running 10.14.6 (Mojave), Xcode 11.2.1

Thanks,

Mike

[CMakeError.log](https://github.com/stitchEm/stitchEm/files/3911298/CMakeError.log)

--- a/doc/BUILD.md
+++ b/doc/BUILD.md
@@ -13,8 +13,9 @@ Using ccache to speed up recompilation is recommended, but not required, on Linu
 sudo port install CMake bison doxygen yasm ninja ccache
 
 # Libraries
-sudo port install opencv glew gsed jpeg libpng \ 
-          tiff faac faad2 ceres-solver glfw ffmpeg glm OpenEXR
+sudo port install opencv glew gsed jpeg libpng \
+          tiff faac faad2 ceres-solver glfw glm OpenEXR \
+          ffmpeg +gpl2 +librtmp +nonfree
 ```
 
 ```

